ttt-online/src/models/game.py

```diff
@@ -118,3 +123,14 @@ def to_dict(self) -> dict:
             if not res[prefix]['id']:
                 res[prefix] = None
         return res
+
+
+class Games(list):
+    def from_record(self, rows: Record) -> Games:
+        games = [dict(row) for row in rows]
+        for game in games:
+            self.append(Game.from_dict(game))
+        return self
+
+    def to_dict(self) -> t.List[dict]:
```

Comment:
What if name it something like in schematics lib, `to_primitive` since it return type is not dict, and not just list.
And it returns primitive types from objects